web/src/utils/LoggedIn.ts

```diff
@@ -0,0 +1,21 @@
+import { useDispatch } from 'react-redux';
```

Comment:
Okay you've set `LoggedIn` for this file name, and the default function is `HandleLoggedIn`. 

Well since it's just a util function, why don't you modify file name to `UserUtils.ts` and combine these two `LoggedIn` and `Loggedout` method in it? And also modify method names to `setUserInformation` and `clearUserInformation` like that.